README.md

```
@@ -55,6 +56,31 @@ Pour performs multi-pass retrieval to build a complete picture:
+```bash
+claude plugin install https://github.com/norrietaylor/distillery
+```
+
+This reads [`plugin.json`](plugin.json) and:
+1. Copies all six skills to `~/.claude/skills/distillery/` (available in every project)
+2. Prompts you to configure the MCP server in `~/.claude/settings.json`
+3. Guides you through setting up your API key
+
+After installation, restart Claude Code and verify with:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)